requirements.txt

```diff
@@ -1,3 +1,4 @@
 click>=6.6
 # PyYAML
 six>=1.10
+requests>=2.25.1
```

Comment:
This probably needs to be version 2.21.0 as this version doesnt seem to work with python 3.4.